### DISCUSSION TITLE
How to override default tags page?

### DISCUSSION THREAD

**Discussion by colemilne54** [chrismwilliams/astro-theme-cactus#81] [Q&A]
I want to make a custom tags page for only my "dev" tag. I want to add some text to the overview of the "dev" tag while still showing all of the posts with the "dev" tag showing up automatically. I tried making a dev folder within the tags directory and created an index.astro file but am having issues removing and adding what is needed.

**Comment by colemilne54** [comment]
I conditionally rendered the relevant content within the [...page].astro when tag === 'dev' but am still curious as to how i can set up a custom tags page that overrides the default one.

**Comment by chrismwilliams** [comment]
Ah I see, yeah the conditional render option is going to be the simplest way, but to set a custom page you can add a `dev.astro` file into the `pages/tags` folder which will override the [priority order](https://docs.astro.build/en/core-concepts/routing/#route-priority-order), or if you prefer `pages/tags/dev/index.astro`

I believe you could also add `pages/tags/dev/[...pages].astro` if you require pagination as well.
